launcher/image/test/test_debug_cloudbuild.yaml

```diff
@@ -39,6 +39,10 @@ steps:
   id: ChangeMDSVariablesTest
   entrypoint: 'bash'
   args: ['scripts/test_mds_var_change.sh', '${_VM_NAME_PREFIX}-${BUILD_ID}', '${_ZONE}']
+- name: 'gcr.io/cloud-builders/gcloud'
+  id: CheckDebugVMAlive
+  entrypoint: 'bash'
+  args: ['scripts/test_debug_alive.sh', '${_VM_NAME_PREFIX}-${BUILD_ID}', '${_ZONE}']
```

Comment:
nit: Consider moving the testing script contents to `args` here, which I think it would be easier to read. No need to create a separate testing shell script.

Reply:
moved, using the "script" field in cloudbuild

Reply:
moved, using the "script" field in cloudbuild